### PR TITLE
Add stable diffusion bf16 inference scripts

### PR DESCRIPTION
**Pull request by mengfei25**
Add stable diffusion bf16 inference scripts on Xeon® with PyTorch and Intel® Extension for PyTorch

### CODE REVIEW

File: examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Prerequisite
+## Create Environment
+```
+conda install mkl mkl-include -y
+conda install jemalloc gperftools -c conda-forge -y
+pip install torch==1.13.1 --extra-index-url https://download.pytorch.org/whl/cpu
+pip install intel_extension_for_pytorch=1.13.0
+pip install -r requirements.txt
+```
+## Setup variable to speedup
```
Comment (hshen14) [inline examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md:16]: Setup Environment Variables

File: examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+export OMP_NUM_THREADS=< Cores to use >
+export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libiomp5.so
+# Jemalloc
+export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libjemalloc.so
+export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:9000000000,muzzy_decay_ms:9000000000"
+```
+
+# Performance
+```bash
+numactl -m <node N> -C <cpu list> python run_tti.py
```
Comment (hshen14) [inline examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md:32]: change to run_sd_bf16.py? do you want to expose precision as a parameter？ We want to add FP32 precision inference also?

File: examples/pytorch/nlp/huggingface_models/text-to-image/inference/run_tti.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+from diffusers import StableDiffusionPipeline, DPMSolverMultistepScheduler
+import torch
+import intel_extension_for_pytorch as ipex
+import time
+
+# load model
+model_id = "runwayml/stable-diffusion-v1-5"
+dpm = DPMSolverMultistepScheduler.from_pretrained(model_id, subfolder="scheduler")
+pipe = StableDiffusionPipeline.from_pretrained(model_id, scheduler=dpm, torch_dtype=torch.float)
+# pipe = pipe.to("cuda")
```
Comment (hshen14) [inline examples/pytorch/nlp/huggingface_models/text-to-image/inference/run_tti.py:11]: remove unused code

File: examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md (hunk 2 above)
Comment (hshen14) [reply examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md:32]: or you can create a separate file run_sd_fp32.py

**Comment by mengfei25** [comment]
Updated, please review.

**Review by hshen14 — APPROVED** [review]

File: examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+The script ```run_sd.py``` is based on [runwayml/stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5) and provides inference benchmarking.
+
+# Prerequisite
+## Create Environment
+```
+conda install mkl mkl-include -y
+conda install jemalloc gperftools -c conda-forge -y
+pip install torch==1.13.1 --extra-index-url https://download.pytorch.org/whl/cpu
+pip install intel_extension_for_pytorch=1.13.0
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md:13]: pip install intel_extension_for_pytorch==1.13.0

**Review by hshen14 — APPROVED** [review]

**Comment by chensuyue** [comment]
Local test pass with BF16/FP32 ~2x perf gain.

**Pull request merged**